File: backend/scripts/inspect_zip.py
import zipfile

with zipfile.ZipFile(r'c:\Website-fixing\electrozonebd_complete_website.zip', 'r') as z:
    names = z.namelist()
    print(f"Total files in zip: {len(names)}")
    critical = [
        'index.html',
        '.htaccess',
        'main.dart.js',
        'flutter.js',
        'api/.env',
        'api/public/index.php',
        'backend/.env',
        'backend/public/index.php'
    ]
    for c in critical:
        status = "OK" if c in names else "MISSING"
        print(f"[{status}] {c}")

File: backend/scripts/inspect_zip.php
<?php
$zip = new ZipArchive();
if ($zip->open('c:\Website-fixing\electrozonebd_complete_website.zip') === TRUE) {
    echo "Total files in zip: " . $zip->numFiles . "\nSample files:\n";
    for ($i = 0; $i < min(25, $zip->numFiles); $i++) {
        echo " - " . $zip->getNameIndex($i) . "\n";
    }
    // Check specific critical files
    $critical = [
        'index.html',
        '.htaccess',
        'main.dart.js',
        'assets/AssetManifest.bin',
        'api/.env',
        'api/public/index.php',
        'backend/.env'
    ];
    echo "\nCritical file checks:\n";
    foreach ($critical as $c) {
        $found = ($zip->locateName($c) !== false);
        echo " [" . ($found ? "OK" : "MISSING") . "] $c\n";
    }
    $zip->close();
} else {
    echo "Failed to open zip\n";
}
